src/config.rs

```diff
@@ -19,6 +19,7 @@ pub struct Config
     pub NHENTAI_TAGS: Option<Vec<String>>, // keep creating downloadme.txt from these tags and keep downloading (server mode), normal tags are in format "tag:{tag}" for example "tag:ffm-threesome"; if None: don't generate downloadme.txt, download hentai once (client mode)
     pub SLEEP_INTERVAL: Option<u64>, // sleep interval in seconds between checking for new hentai to download (server mode)
     pub USER_AGENT: Option<String>, // bypass bot protection
+    pub ARCHIVE_ORG: Option<bool>, // Allow pull from archive.org? False by default
```

Comment:
rename to `FALLBACK_TO_ARCHIVE_ORG`, insert at a line that retains alphabetical order with other settings, lowercase comments

Reply:
Done.